### ISSUE TITLE
Set $USER variable by default

### ISSUE THREAD

**Issue by felipecassiors** [microsoft/vscode-dev-containers#490]
Start a container and you'll note that the `$USER` environment variable won't be set. This variable should be injected in the container by VSCode since it knows which is user name.

In my case, I use this environment variable in my `dotfiles/install` script.

**Comment by Chuxel** [comment]
Yeah, script-library/common-debian.sh should be able to add this via .bashrc/.zshrc using the id command.

**Comment by felipecassiors** [comment]
@Chuxel that would not solve the problem, since `.bashrc` and `.zshrc` are replaced when using `dotfiles`.

We need to find something different.

**Comment by Chuxel** [comment]
That's then a feature request for the extension.  Will move to the other repo. //cc: @chrmarti
